api/data/it/PD.tsx

```diff
@@ -108,13 +108,16 @@ export const Cards1 = () => {
         <Box pb={8} pt={8}>
             <Container maxWidth="xl" className="documentiCustomCard">
                 <Grid container spacing={2}>
-                    <Grid item xs={12} md={4} pb={4} sx={{ pr: { md: 10, xs: 0, sm: 0 } }}>
+
```

Comment:
@salvatorediocaro consider to remove grid as layout system and using Stack https://mui.com/material-ui/react-stack/ and flexbox to adjust this page layout. 
Using fake columns to diplay items is not advised